output-data/version.txt

```diff
@@ -1,9 +1,9 @@
-ClientVersion=5226
-ServerVersion=5226
+ClientVersion=5220
+ServerVersion=5220
 ProductName=citadel
 appID=1422450
 ServerAppID=1422460
 ToolsAppID=211
-SourceRevision=9219272
-VersionDate=Sep 30 2024
-VersionTime=11:12:14
+SourceRevision=9217762
+VersionDate=Sep 27 2024

```

Comment:
I put the new patch on develop so should reparse and then its good to go!

Reply:
I already reparsed, looks like it matches develop

Reply:
this is a diff between your branch and develop. Maybe steam needs to update the game?